src/faultlog/util.cpp

```diff
@@ -64,18 +56,10 @@ HostState getHostState(sdbusplus::bus::bus& bus)
 {
     try
     {
-        using PropertiesVariant =
-            sdbusplus::utility::dedup_variant_t<HostState>;
-
-        auto retVal = readProperty<PropertiesVariant>(
-            bus, "xyz.openbmc_project.State.Host",
-            "/xyz/openbmc_project/state/host0",
-            "xyz.openbmc_project.State.Host", "CurrentHostState");
-        const HostState* statePtr = std::get_if<HostState>(&retVal);
-        if (statePtr != nullptr)
-        {
-            return *statePtr;
-        }
+        return readProperty<HostState>(bus, "xyz.openbmc_project.State.Host",
```

Comment:
Do we need to fix the readProperty here? https://github.com/ibm-openbmc/openpower-hw-isolation/blob/658b2a036959dea9dbcc3ef107dbf7fe770db750/src/faultlog/faultlog_main.cpp#L186